keyboards/smithrune/iron180/v2s/config.h

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2023 Gondolindrim  <gondolindrim@acheronproject.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define BACKLIGHT_PWM_DRIVER    PWMD3
+#define BACKLIGHT_PWM_CHANNEL   3
+#define BACKLIGHT_PAL_MODE      2
+#define BACKLIGHT_CAPS_LOCK
```

Comment:
```suggestion
```

Reply:
Apart from line 23, I think the rest shouldn't be removed because V2S has backlight

Reply:
https://github.com/qmk/qmk_firmware/blob/develop/docs/feature_backlight.md#pwm-driver-idarm-pwm-driver

Reply:
Not sure what the suggestion is, the configuration is there

Reply:
`BACKLIGHT_PWM_CHANNEL` and `BACKLIGHT_PAL_MODE` are the default.